src/app/component/layout/components/footer/footer.component.html

```diff
@@ -3,61 +3,65 @@
-            </li>
-            <li>
-              <a [routerLink]="['/profile', getUserId()]" class="footer_link-item">
-                {{ "user.footer.my-habits" | translate }}
-              </a>
-            </li>
-          </ul>
+      <nav class="footer_links" aria-label="Footer Navigation Menu">
+        <ul class="footer_left-side">
+          <li>
+            <a [routerLink]="['/news']" class="footer_link-item">
+              {{ "user.footer.eco-events" | translate }}
+            </a>
+          </li>
+          <li>
+            <a [routerLink]="['/tips']" class="footer_link-item">
+              {{ "user.footer.tips" | translate }}
+            </a>
+          </li>
+          <li>
+            <a [routerLink]="['/map']" class="footer_link-item">
+              {{ "user.footer.map" | translate }}
+            </a>
+          </li>
+          <li>
+            <a [routerLink]="['/about']" class="footer_link-item">
+              {{ "user.footer.about-us" | translate }}
+            </a>
+          </li>
+          <li>
+            <a
+              [routerLink]="['/profile', getUserId()]"
+              class="footer_link-item"
+            >
+              {{ "user.footer.my-habits" | translate }}
+            </a>
+          </li>
+        </ul>
 
-          <ul class="footer_right-side">
-            <li class="footer_follow-us">
-              <p><span>{{ "user.footer.follow-us" | translate }}</span></p>
-            </li>
-            <li class="footer_social-wrp">
-              <a class="footer_social-link" href="#">
-                <img [src]="footerImageList.twitterIcon"
-                      alt="Twitter link">
-              </a>
-              <a class="footer_social-link" href="#">
-                <img [src]="footerImageList.linkedInIcon"
-                      alt="LinkedIn link">
-              </a>
-              <a class="footer_social-link" href="#">
-                <img [src]="footerImageList.facebookIcon"
-                      alt="Facebook link">
-              </a>
-            </li>
+        <ul class="footer_right-side">
+          <li class="footer_follow-us">
+            <p>
+              <span>{{ "user.footer.follow-us" | translate }}</span>
+            </p>
+          </li>
+          <li class="footer_social-wrp">
+            <a class="footer_social-link" href="#">
+              <img [src]="footerImageList.twitterIcon" alt="Twitter link" />
+            </a>
+            <a class="footer_social-link" href="#">
+              <img [src]="footerImageList.linkedInIcon" alt="LinkedIn link" />
+            </a>
+            <a class="footer_social-link" href="#">
+              <img [src]="footerImageList.facebookIcon" alt="Facebook link" />
+            </a>
+          </li>
         </ul>
-        </nav>
-      <div class="footer_bottom-part">
+      </nav>
+      <div class="footer_bottom-part" id="copyright-label">
         &copy; Copyright {{ actualYear }}. Green City.
       </div>
```

Comment:
![image](https://user-images.githubusercontent.com/24696596/113360499-961cc000-9352-11eb-8f59-188cd6eb4334.png)
It looks not good in my display.
Why you used Id? Maybe class better.

Reply:
I have added additional rule to make sure it will always display without any wapping. Commit b1d33375a302c4ae82261b02bf302ead5bddd70c

ID has been used instead of Class because class leads to additional styling work. So I've chosen a one-line solution